appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -70,16 +83,25 @@ private fun Modifier.styleBackground(
     isDark: Boolean,
     defaultColor: Color? = null
 ) = this.then(
+    Modifier
```

Comment:
I made this change so its in sync with IOS, instead of picking one of the two we now apply whatever we have in sequence to make sure we can mix those properties if we want to.

also this was supposed to have a 3rd modifier called styleBackgroundImage, but this is where I could not make it work  the way I wanted to, still good change to match IOS behavior.